Documentation/operations/performance/tuning.rst

```diff
@@ -175,6 +175,16 @@ To enable eBPF Host-Routing:
              --set bpf.masquerade=true \\
              --set kubeProxyReplacement=true
 
+**Known limitations:**
+
+eBPF host routing optimizes the host-internal packet routing, and packets no
+longer bypass the netfilter in the host namespace. Therefore, it is incompatible
```

Comment:
Isn't it the other way around?

```suggestion
eBPF host routing optimizes the host-internal packet routing, and packets
bypass the netfilter in the host namespace. Therefore, it is incompatible
```

Or

```suggestion
eBPF host routing optimizes the host-internal packet routing, and packets no
longer hit the netfilter tables in the host namespace. Therefore, it is incompatible
```

Probably some variant on the latter is better for general readers to avoid confusion about "bypass" being to "avoid" or to "pass by" which have opposite meaning.

Reply:
It is! Thanks!